dev/tasks/r/github.packages.yml

```diff
@@ -280,10 +319,33 @@ jobs:
         with:
           install-r: false
       {{ macros.github_setup_local_r_repo(false, false)|indent }}
+      - name: Get sccache
+        shell: bash
+        run: |
+          SCCACHE_BUILD=unknown-linux-musl
+          SCCACHE_URL="https://github.com/mozilla/sccache/releases/download/v0.3.0/sccache-v0.3.0-x86_64-$SCCACHE_BUILD.tar.gz"
+
+          # Download archive and checksum
+          curl -L $SCCACHE_URL --output sccache.tar.gz
+          curl -L $SCCACHE_URL.sha256 --output sccache.tar.gz.sha256
+
+          SCCACHE_ARCHIVE=sccache.tar.gz
+          echo "$(cat $SCCACHE_ARCHIVE.sha256) $SCCACHE_ARCHIVE" | sha256sum --check --status
+
+          mkdir -p sccache
+          tar -xzvf $SCCACHE_ARCHIVE --strip-component=1 --directory sccache
+
+          echo "$(pwd)/sccache" >> $GITHUB_PATH
+          echo "SCCACHE_PATH=$(pwd)/sccache/sccache.exe" >> $GITHUB_ENV
+      - run: sudo apt update && sudo apt install libcurl4-openssl-dev

```

Comment:
why is this now required? Should be done on the previous step (install sscache)?

Reply:
Seems a bit weird indeed. Why would dev dependencies be neede for a compiled executable?

Reply:
This is a source build and does not use the binaries, so these are required for aws/gcs bindings to be enabled.